back-end/server.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const UserRouter = require('./routes/UserRoutes');
+const { default: mongoose } = require('mongoose');
```

Comment:
what you used default:mongoose instead of only mongoose

Reply:
Vs code did it on its own, I did not even know it was there. I apologise